closed/OpenJ9.gmk

```diff
@@ -63,6 +63,13 @@ else
   OMR_EXTRA_CONFIGURE_ARGS := "--enable-warnings-as-errors OMR_TREAT_WARNINGS_AS_ERRORS=1"
 endif
 
+# disable mingw on windows, use default c compiler
+ifeq (windows,$(OPENJDK_TARGET_OS))
```

Comment:
Can you reverse the argument order for consistency with other such tests?
```
ifeq ($(OPENJDK_TARGET_OS), windows)
```

Reply:
Fixed as suggested, thank you!